### ISSUE TITLE
高占用内存

### ISSUE THREAD

**Issue by luxuefengGIT** [microsoft/vscode#73456]
Issue Type: <b>Performance Issue</b>

code helper 高占用内存

VS Code version: Code 1.33.1 (51b0b28134d51361cf996d2f0a1c698247aeabd8, 2019-04-11T08:22:55.268Z)
OS version: Darwin x64 18.2.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz (4 x 2300)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 18, 16|
|Memory (System)|16.00GB (5.17GB free)|
|Process Argv|-psn_0_61455|
|Screen Reader|no|
|VM|0%|

</details><details>
<summary>Process Info</summary>

```
CPU %	Mem MB	   PID	Process
   20	   131	   281	code main
    2	    98	   419	   gpu-process
    0	   131	   432	   shared-process
    0	   295	 20754	   window (leavatask.js — leave-task)
    0	   229	 20759	     extensionHost
    0	   229	 20763	       electron_node tsserver.js 
    0	    66	 20793	         electron_node typingsInstaller.js typesMap.js 
    0	    49	 20767	       electron_node server.js 
    0	   213	 20797	       electron_node server.js 
    0	    49	 20871	       /Users/luxuefeng/Desktop/Visual Studio Code.app/Contents/Frameworks/Code Helper.app/Contents/MacOS/Code Helper /Users/luxuefeng/Desktop/Visual Studio Code.app/Contents/Resources/app/extensions/json-language-features/server/dist/jsonServerMain --node-ipc --clientProcessId=20759
    0	    49	 20760	     watcherService
    0	    49	 20766	     searchService
    6	    82	 20915	   window (问题报告程序)
    3	     0	 20919	   /bin/ps -ax -o pid=,ppid=,pcpu=,pmem=,command=
```

</details>
<details>
<summary>Workspace Info</summary>

```
|  Window (leavatask.js — leave-task)
|    Folder (leave-task): 28 files
|      File types: js(13) DS_Store(3) css(2) html(2) json(2) eslintrc(1)
|                  gitkeep(1) gitignore(1) editorconfig(1) less(1)
|      Conf files: package.json(1);
```

</details>
<details><summary>Extensions (22)</summary>

Extension|Author (truncated)|Version
---|---|---
html-snippets|abu|0.2.1
color-info|bie|0.5.1
npm-intellisense|chr|1.3.0
path-intellisense|chr|1.4.2
bracket-pair-colorizer|Coe|1.0.61
gitlens|eam|9.7.1
vscode-html-css|ecm|0.2.0
auto-close-tag|for|0.5.6
auto-rename-tag|for|0.0.15
beautify|Hoo|1.5.0
vscode-gutter-preview|kis|0.19.0
vscode-language-pack-zh-hans|MS-|1.33.2
debugger-for-chrome|msj|4.11.3
vetur|oct|0.20.0
vscode-css-peek|pra|2.2.0
LiveServer|rit|5.6.1
html5-boilerplate|sid|1.0.3
open-in-browser|tec|2.0.0
vscode-icons|vsc|8.6.0
vscode-react-native|vsm|0.9.2
quokka-vscode|Wal|1.0.212
html-css-class-completion|Zig|1.18.0

</details>

**Comment by chrmarti** [comment]
Please see https://github.com/Microsoft/vscode/wiki/Performance-Issues for providing additional information on performance issues.

Please write in English.

**Comment by luxuefengGIT** [comment]
thank you. 


 
---原始邮件---
发件人: "vscodebot[bot]"<notifications@github.com>
发送时间: 2019年5月8日(星期三) 晚上11:50
收件人: "microsoft/vscode"<vscode@noreply.github.com>;
抄送: "Author"<author@noreply.github.com>;"luxuefengGIT"<10348051@qq.com>;
主题: Re: [microsoft/vscode] 高占用内存 (#73456)



Closed #73456.
